golangci.yaml

```diff
@@ -1,12 +1,35 @@
-linters:
-  disable:
-    - megacheck
+liners:
+  disable-all: true
   enable:
+    - bodyclose
+    - deadcode
+    - depguard
+    - dogsled
+    - errcheck
+    - gocyclo
+    - gofmt
+    - goimports
+    - goprintffuncname
+    - gosec
     - gosimple
+    - govet
+    - ineffassign
+    - interfacer
+    - nakedret
+    - nolintlint
+    - rowserrcheck
```

Comment:
`rowserrcheck` is not applicable in this repo, as we don't make use of golang's `sql` package.

Reply:
+1